### PR TITLE
Fix code examples in "Manage Qiskit Serverless compute and data resources"

### PR DESCRIPTION
**Pull request by frankharkins**
This makes the code examples runnable. Unfortunately we can't add tests just yet (will happen in #1960), but this does mean users will be able to run all the code on the page.

I've removed the section on automatic QPU selection after discussion with the serverless team. This feature is being developed but isn't ready just yet. We'll add the section back in when it's ready.

### CODE REVIEW

**Comment by frankharkins** [comment]
PR did deploy, check out the page here: https://docs.quantum.ibm.com/guides/serverless-manage-resources

File: docs/guides/serverless-manage-resources.ipynb [hunk 1]
```diff
@@ -108,81 +126,16 @@
    "metadata": {},
    "outputs": [],
    "source": [
+    "%%writefile --append ./source_files/transpile_remote.py\n",
+    "\n",
     "@distribute_task(target={\n",
     "    \"cpu\": 16,\n",
-    "    \"mem\": 32 * 1024 * 1024 * 1024\n",
+    "    \"mem\": 2 * 1024 * 1024 * 1024\n",
```
Comment (Eric-Arellano) [inline docs/guides/serverless-manage-resources.ipynb:133]: Why this change?

**Review by Eric-Arellano — APPROVED** [review]
Thanks!

File: docs/guides/serverless-manage-resources.ipynb (hunk 1 above)
Comment (frankharkins) [reply docs/guides/serverless-manage-resources.ipynb:133]: The prose says:

>  To allocate 2 GB of memory for your worker, you need to allocate `"mem": 2 * 1024 * 1024 * 1024`

As a reader, I was mildly surprised when the code example allocated 32GB rather than 2.
